packages/core/core/src/types.ts

```diff
@@ -255,6 +254,12 @@ export interface AgnosticCategoryTree {
   [x: string]: unknown;
 }
 
+export interface AgnosticCoupon {
+  id: string;
+  name: string;
+  code: string;
```

Comment:
please add `value` and set it to null in CT for now, but please create an issue for that, to implement that field in CT

Reply:
Done